logic.js

```diff
@@ -0,0 +1,187 @@
+stored_posts =
+[
+    {
+        id: 1,
+        text: "First post!"
+    },
+    {
+        id: 2,
+        text: "Second post!"
+    }
+];
+
+stored_comments =
```

Comment:
in JS the standard is to camelCase your variables. We never use `_` between words in our variables in JS.

Reply:
thanks alot :) I will improve my code next time